app/pyproject.toml

```diff
@@ -40,6 +40,8 @@ pytest-watch = "^4.2.0"
 pytest-lazy-fixture = "^0.6.3"
 types-pyyaml = "^6.0.12.11"
 setuptools = "^68.2.2"
+sadisplay = "0.4.9"
```

Comment:
[looks like this tool is archived](https://github.com/GoodRx/sadisplay)
<img width="1163" alt="image" src="https://github.com/navapbc/template-application-flask/assets/447859/5f4b2cac-0e70-4437-8832-4baaca374e74">

[eralchemy](https://github.com/Alexis-benoist/eralchemy) looks like it has a lot more stars/forks, and the diagrams look better.

Taking a step back, we'll probably need something like this regardless of the tech stack so that we don't have to maintain two separate implementations for python and next.js templates (and in the future we'll probably have a java template too), so I'd like to explore a language-agnostic one that runs in the infra template.

From browsing [this reddit post](https://www.reddit.com/r/PostgreSQL/comments/un877d/generate_erd_from_existing_database_schema_what/), it seems that https://github.com/schemaspy/schemaspy is pretty nice. I like that the generated documentation is also interactive. For example, the diagram we have on MA PFML is pretty impossible to navigate and read since it's so big, so having interactive features could be really nice. Thoughts?

Reply:
Of course it was archived 4 days after I added it to the Grants repo - If I've got time I'll look into Schema spy in the near future